scripts/secret-creation/secret-creation.sh

```diff
@@ -55,12 +55,12 @@ while [[ $# -gt 0 ]]; do
     shift
 done
 
-[ -z ${service} ] && service=mutatepodtoleration
-[ -z ${secret} ] && secret=pod-mutator-secret
-[ -z ${namespace} ] && namespace=default
-[ -z ${outputdir} ] && outputdir=/etc/ssl/liqo
-[ -z ${outputenvfile} ] && outputenvfile=/etc/environment/liqo
-[ -z ${exportenvvars} ] && exportenvvars=false
+[ -z "${service}" ] && service=mutatepodtoleration
```

Comment:
Could be replaced with the bash *default* syntax, which is more concise (applies to all similar lines)